### DISCUSSION TITLE
Releases of [@types/node] are too often

### DISCUSSION THREAD

**Discussion by eirnym** [DefinitelyTyped/DefinitelyTyped#68434] [Issues with a @types package]
Hi, 

in past few days releases rate is too high. I've got quite a few PRs that `@types/node` library has been upgraded (patch release).
Could you please to release them less often and please add tags to makes others be sure what commit is belong to what release.

[a comment by an automated account is not shown]

**Comment by yoursunny** [comment]
Releases are automated upon merging a Pull Request.
You can choose to upgrade monthly and don't have to upgrade every time.

**Reply by eirnym** [reply]
Could you please document update policy for everybody to make sure that everybody has the same info?

**Reply by eirnym** [reply]
What is the policy for minor updates?

**Reply by Semigradsky** [reply]
See https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library

**Reply by eirnym** [reply]
Little bit of a background on my problem: I have RenovateBot working on updates and don't do manual package updates. Every single update is a change of yarn.lock file and is a potential conflict, which leads to delays in development. 

Linked readme file describes automatic patch (currently they're patch as I see PRs from RenovateBot) and major updates with breaking changes. What's about minor updates? How they're handled? Are they threaten as "breaking"? Are they automatic or manual? 

I can disable patch updates but I'll miss all updates beside breaking ones according to readme. And I don't like this idea as it's one of base packages which I can't ignore or exclude

**Reply by yoursunny** [reply]
`@types/*` packages do not follow SemVer at all. Breaking change may occur at any time.
You should thoroughly test the upgrade before applying.
